### PR TITLE
Add medical HUDs to startup loadout for medical doctors

### PR DESCRIPTION
**Pull request by Puckaboo2**
:cl:Puckaboo2
Add: Medical HUD on job spawn for Medical Doctors and the CMO.
Add: Security Sunglasses on job spawn for Sec officers and pod pilots.
/:cl:

### CODE REVIEW

File: code/game/jobs/job/medical.dm [hunk 1]
```diff
@@ -71,14 +72,17 @@
...
 					H.equip_or_collect(new /obj/item/clothing/under/rank/medical/blue(H), slot_w_uniform)
 					H.equip_or_collect(new /obj/item/clothing/suit/storage/labcoat(H), slot_wear_suit)
 					H.equip_or_collect(new /obj/item/clothing/head/surgery/blue(H), slot_head)
 				if("Medical Doctor")
+					H.equip_or_collect(new /obj/item/clothing/glasses/hud/medical(H), slot_glasses)
 					H.equip_or_collect(new /obj/item/clothing/under/rank/medical(H), slot_w_uniform)
 					H.equip_or_collect(new /obj/item/clothing/suit/storage/labcoat(H), slot_wear_suit)
 				if("Nurse")
 					if(H.gender == FEMALE)
+					H.equip_or_collect(new /obj/item/clothing/glasses/hud/medical(H), slot_glasses)
```
Comment (Twinmold93) [inline code/game/jobs/job/medical.dm]: Needs to be moved up a line with this same tabbing.

File: code/game/jobs/job/medical.dm [hunk 2]
```diff
@@ -63,6 +63,7 @@
 		if(H.mind.role_alt_title)
 			switch(H.mind.role_alt_title)
 				if("Coroner")
+					H.equip_or_collect(new /obj/item/clothing/glasses/hud/medical(H), slot_glasses)
```
Comment (Twinmold93) [inline code/game/jobs/job/medical.dm]: Unnecessary with putting line prior.

File: code/game/jobs/job/medical.dm [hunk 3]
```diff
@@ -71,14 +72,17 @@
 						H.equip_or_collect(new /obj/item/device/mass_spectrometer(H), slot_in_backpack)
 						H.equip_or_collect(new /obj/item/weapon/storage/box/bodybags(H), slot_in_backpack)
 				if("Surgeon")
+					H.equip_or_collect(new /obj/item/clothing/glasses/hud/medical(H), slot_glasses)
```
Comment (Twinmold93) [inline code/game/jobs/job/medical.dm]: Unnecessary with putting line prior.

File: code/game/jobs/job/medical.dm [hunk 4]
```diff
@@ -71,14 +72,17 @@
 						H.equip_or_collect(new /obj/item/device/mass_spectrometer(H), slot_in_backpack)
 						H.equip_or_collect(new /obj/item/weapon/storage/box/bodybags(H), slot_in_backpack)
 				if("Surgeon")
+					H.equip_or_collect(new /obj/item/clothing/glasses/hud/medical(H), slot_glasses)
 					H.equip_or_collect(new /obj/item/clothing/under/rank/medical/blue(H), slot_w_uniform)
 					H.equip_or_collect(new /obj/item/clothing/suit/storage/labcoat(H), slot_wear_suit)
 					H.equip_or_collect(new /obj/item/clothing/head/surgery/blue(H), slot_head)
 				if("Medical Doctor")
+					H.equip_or_collect(new /obj/item/clothing/glasses/hud/medical(H), slot_glasses)
```
Comment (Twinmold93) [inline code/game/jobs/job/medical.dm]: Unnecessary with putting line prior.

**Comment by Fox-McCloud** [comment]
I don't think docs should start off with this, personally; you could make the argument to increase the amount on the map, but I don't think they should start out with them.

**Comment by Twinmold93** [comment]
The only reason one I could think to not have them start off with them is so they don't immediately start doctoring until they are at medical and hopefully gear up.  However, if issues ensue (like converted science, explosion in R&D, no materials, etc) and a lack of HUDs/they are all in use already, new doctors (join round doctors) are S.O.L.

This would be more than anything an ease of access/use feature for something that isn't really game altering (unlike security HUDs, which security that join already start off with plus spare in the locker room).

**Comment by DesolateG** [comment]
Security doesn't start with HUD sunglasses; you have to get them from the locker room.

**Comment by Davdichan** [comment]
@Fox-McCloud @DesolateG Security definitely starts with sechuds on when they join. If you don't believe me play a round as sec.

**Comment by DesolateG** [comment]
Unless this was a recent change, everytime I have played security officer and sec pod pilot I have had to get sec Hud glasses from the locker, I did not start with them.

**Comment by Spacemanspark** [comment]
Security officers on /tg/ start with secHUDS, we don't.
That said, I think they should.

**Comment by Puckaboo2** [comment]
I'm adding them to the loadout not just for medical personnel who spawn next to the lockers at startup and are free to raid the lockers, but also for medical personnel who are new to the station, all the way near arrivals without access to medical supplies, perhaps during an emergency. 

In fact, I need to give the CMO more love, too.

**Comment by DesolateG** [comment]
Or give cmo med Hud implant

**Comment by Puckaboo2** [comment]
Or the advanced medical HUD with built in shades :o

**Comment by DesolateG** [comment]
Heh.. funny

**Comment by Doukhan** [comment]
Not really a fan of these sorts of changes, it locks more equipment behind a spawn in gate and takes a lot away from the importance of storage areas/lockers.

**Pull request closed without merging**